packages/x-components/src/x-modules/related-tags/components/related-tags.vue

```diff
@@ -112,9 +112,9 @@ To use this component, the QuerySignals microservice must be implemented.
 
 This example shows how related tags can be rendered without any additional effects.
 
-_Search for a toy and press enter._
+_Search for a fashion term like "sandal" or "lipstick"._
 
-```vue
+```vue live
 <template>
   <div>
     <SearchInput></SearchInput>
```

Comment:
I just found this, all the components that are not using the slots the have to be self-closed: https://v2.vuejs.org/v2/style-guide/?redirect=true#Self-closing-components-strongly-recommended

So please, check all the files that you touched and change the components that can be self-closed to be self-closed.

Reply:
I searched through the examples and changed to self-closed some components in `related-tags`, `related-tag` and `history-queries`.